### PR TITLE
User story #7 root url

### PR DESCRIPTION
**Pull request by mpourostad**
The root URL displays useful information about the service in JSON format. The tests for the homepage should pass when the URL "http://localhost/products" is created and the "paths" in the index function is changed from paths=url_for("index", _external=True) to paths=url_for("list_products", _external=True)

### CODE REVIEW

File: service/routes.py [hunk 1]
```diff
@@ -25,7 +25,9 @@
 def index():
     """ Root URL response """
     return (
-        "Reminder: return some useful information in json format about the service here",
+        # "Reminder: return some useful information in json format about the service here",
+        jsonify( name = "products", paths=url_for("index", _external=True),
```
Comment (yuluqinn) [inline service/routes.py:29]: May change to "Product REST API Service"

**Review by yuluqinn — APPROVED** [review]
Name could be changed to "Product REST API Service"

**Comment by yuluqinn** [comment]
> The root URL displays useful information about the service in JSON format. The tests for the homepage should pass when the URL "http://localhost/products" is created and the "paths" in the index function is changed from paths=url_for("index", _external=True) to paths=url_for("list_products", _external=True)

Have you changed the url from "index" to "list_products" already? I didn't see the changes, am I missing anything?

File: service/routes.py (hunk 1 above)
Comment (mpourostad) [reply service/routes.py]: You're right. I updated the files.

**Pull request merged**